lib/nsxv.py

```diff
@@ -353,6 +374,18 @@ def __str__(self):
     ret_str.extend(stripped_ret_lines)
     return ''.join(ret_str)
 
+  def _DropMismatchIpAddress(self, ipaddress, direction_address, af):
```

Comment:
Please add a docstring to this function if you intend to keep it. Look at other functions for examples. It should have a sentence description saying what it does and a section explianing arguments and returns and raises